index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Inter">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Benjamin Patin's Developer Portfolio</title>
+  </head>
+  <body>
+
+    <!--- Top navigation Bar --->
+    <div class="topnav">
+      <div class="left"></div>
+      <div class="nav-menu">
+        <a href="#news">Portfolio</a>
+        <a href="#about">About</a>
+        <a href="#contact">Contact</a>
+      </div>
+      <div class= "right"></div>
+    </div>
+
+  <!--- Headline Section --->
+    <div class="headline">
+      <div class="headline-text1">Hey There, I am Benjamin Patin.</div>
+      <div class="headline-text2">I am a Full Stack developper.</div>
```

Comment:
- [OPTIONAL] I think you could use the `<h2>` tag for this header since you could use the `<h1>` for the site title/logo (and there could only be 1 `<h1>` tag per page). Remember that the most probable is that there is a replacement for a `<div>` tag :+1: